sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/cases/CaseImportExportSteps.java

```diff
@@ -106,6 +124,16 @@ public CaseImportExportSteps(
           webDriverHelpers.clickOnWebElementBySelector(NEW_EXPORT_CONFIGURATION_SAVE_BUTTON);
         });
 
+    And(
+        "I add ([^\"]*) data to export in existing Export Configuration for Custom Case Export",
+        (String customExportConfigurationCheckbox) ->
+            webDriverHelpers.clickOnWebElementBySelector(
+                getCustomExportCheckboxByText(customExportConfigurationCheckbox)));
+
+    Then(
+        "I save Custom Export Configuration",
```

Comment:
rename it with the action performed in UI,  "I click on Save button from {page location}"